web/package.json

```diff
@@ -21,10 +21,11 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
     "recoil": "^0.1.2",
+    "swr": "^0.5.3",
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "start:dev": "react-scripts start",
+    "start:dev": "PORT=3001 react-scripts start",
```

Comment:
`PORT` is set in `.env.example` now, so I don't think this is needed, right?

Reply:
It wasn't picking it up for some reason. I'll double check.